Corgibytes.Freshli.Cli/OutputFormatter.cs

```diff
@@ -1,31 +1,40 @@
-using Freshli;
+using Corgibytes.Freshli.Lib;
 using System.Collections.Generic;
+using System.Globalization;
 using System.IO;
+using Corgibytes.Freshli.Cli.Resources;
 
-namespace Corgibytes.Freshli.Cli {
-  public class OutputFormatter {
-    private readonly TextWriter _writer;
+namespace Corgibytes.Freshli.Cli
+{
+    public class OutputFormatter
+    {
+        private const string Separator = "\t";
+        private readonly TextWriter _writer;
 
-    public OutputFormatter(TextWriter writer) {
-      _writer = writer;
-    }
+        public OutputFormatter(TextWriter writer)
+        {
+            _writer = writer;
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture; //format for dates and numbers to use invariant
```

Comment:
It should be documented in the ReadMe that we use the Invariant Culture, at least for now.